src/main/java/net/modfest/scatteredshards/ShardCommand.java

```diff
@@ -150,8 +150,18 @@ public static int deleteAll(CommandContext<ServerCommandSource> ctx) throws Comm
 	}
 	
 	public static CompletableFuture<Suggestions> suggestShardIds(CommandContext<ServerCommandSource> source, SuggestionsBuilder builder) {
+		String prefix = builder.getRemaining();
+		for(Identifier id : ScatteredShardsComponents.getShardLibrary(source).getShardIds()) {
+			if (prefix == "" || id.toString().startsWith(prefix)) builder.suggest(id.toString());
```

Comment:
Should these `== ""` checks be `.isEmpty()` or `.isBlank()`?

Reply:
Good catch, should be isBlank